### PR TITLE
Replace hardcoded strings with environment variables

### PR DESCRIPTION
**Pull request by lappalj4**
Moved here from #40

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -30,24 +30,24 @@
...
-    <meta name="twitter:image" content="https://res.cloudinary.com/hackjunction/image/upload/c_fill,g_center,h_600,w_1200/rqn6uonv2nuzmkyanrqj" />
+    <meta property="og:image" content="%REACT_APP_SEO_IMAGE_URL%"/>
+    <meta name="twitter:image" content="%REACT_APP_SEO_IMAGE_URL%" />
     <meta property="og:image:width" content="1200"/>
     <meta property="og:image:height" content="630"/>
     <meta name="twitter:card" content="summary_large_image">
-    <meta name="twitter:site" content="@hackJunction">
-    <meta name="twitter:creator" content="@hackJunction">
+    <meta name="twitter:site" content="%REACT_APP_SEO_TWITTER_HANDLE%">
+    <meta name="twitter:creator" content="%REACT_APP_SEO_TWITTER_HANDLE%">
```
Comment (lappalj4) [inline frontend/public/index.html]: So for this to work, we'll need to make the following additions to the `.env` for all deployments:

```
REACT_APP_SEO_PAGE_TITLE=Experience Europe’s Leading Hackathon
REACT_APP_SEO_PAGE_DESCRIPTION=Organized in the Helsinki Area, Finland, Junction is a meeting place for thousands of developers, designers, and entrepreneurs. A weekend-long experience, gathering tech enthusiasts from all over the world to create with the latest technology in a unique environment and atmosphere.
REACT_APP_SEO_IMAGE_URL=https://res.cloudinary.com/hackjunction/image/upload/c_fill,g_center,h_630,w_1200/rqn6uonv2nuzmkyanrqj
REACT_APP_SEO_TWITTER_HANDLE=@hackJunction
```

File: frontend/src/components/navbars/GlobalNavBar/index.js [hunk 2]
```diff
@@ -32,7 +33,7 @@ export default () => {
                 <img
                     src={require('assets/logos/wordmark_black.png')}
                     className={classes.wordmark}
-                    alt="Junction wordmark"
+                    alt={config.PLATFORM_OWNER_NAME + ' wordmark'}
                 />
                 <UserMenu />
```
Comment (lappalj4) [inline frontend/src/components/navbars/GlobalNavBar/index.js]: Let's change this to use a url, so we can customise the logo without causing a git diff.

File: frontend/src/constants/config.js [hunk 3]
```diff
@@ -13,6 +13,10 @@ const settings = {
         value: process.env.REACT_APP_AUTH0_CLIENT_ID,
         required: true,
     },
+    ID_TOKEN_NAMESPACE: {
+        value: process.env.REACT_APP_ID_TOKEN_NAMESPACE,
+        required: true,
+    },
```
Comment (lappalj4) [inline frontend/src/constants/config.js]: Needs to be changed to default to what it was before

File: frontend/src/pages/_account/profile/index.js [hunk 4]
```diff
@@ -217,7 +218,8 @@ export default () => {
                                             Your contact email address, where
                                             you want to receive necessary
                                             notifications related to your
-                                            activity on the Junction app. Your
+                                            activity on the{' '}
+                                            {config.PLATFORM_OWNER_NAME} app.
                                             email address will never be shared
```
Comment (lappalj4) [inline frontend/src/pages/_account/profile/index.js]: Missing a few words here

File: frontend/src/pages/_account/profile/index.js [hunk 5]
```diff
@@ -303,16 +305,16 @@ export default () => {
...
+                                When you register to events on the{' '}
+                                {config.PLATFORM_OWNER_NAME} app, the below
+                                details will be pre-filled into your
                                 registrations, and updated from your latest
                                 registration. In case you have opted in for
                                 recruitment functionality, these details will
-                                also be shown to select Junction partners who
+                                also be shown to select{' '}
+                                {config.PLATFORM_OWNER_NAME} partners who are
                                 are looking to hire. Please see our{' '}
```
Comment (lappalj4) [inline frontend/src/pages/_account/profile/index.js]: Typo

File: frontend/src/pages/_index/index.js [hunk 6]
```diff
@@ -37,11 +38,11 @@ export default () => {
...
+                        <h2>New to {config.PLATFORM_OWNER_NAME}?</h2>
                         <p>
-                            More info about Junction can be found from our
-                            website{' '}
-                            <ExternalLink href="https://hackjunction.com">
+                            More info about {config.PLATFORM_OWNER_NAME} can be
+                            found from our website{' '}
+                            <ExternalLink href={config.PLATFORM_OWNER_WEBSITE}>
                                 {' '}
                                 here
```
Comment (lappalj4) [inline frontend/src/pages/_index/index.js]: Extra space here

File: frontend/src/pages/_login/welcome/index.js [hunk 7]
```diff
@@ -180,7 +181,7 @@ export default () => {
                     <img
                         className={classes.contentLeftLogo}
                         src={require('assets/logos/wordmark_white.png')}
-                        alt="Junction logo"
+                        alt={config.PLATFORM_OWNER_NAME + ' logo'}
                     />
                 </div>
```
Comment (lappalj4) [inline frontend/src/pages/_login/welcome/index.js]: Change this to use a url as well

File: frontend/src/pages/_login/welcome/index.js [hunk 8]
```diff
@@ -297,7 +298,7 @@ export default () => {
                                         </a>{' '}
                                         and{' '}
                                         <a
-                                            href="https://hackjunction.com/policy"
```
Comment (lappalj4) [inline frontend/src/pages/_login/welcome/index.js]: Some extra whitespace here

File: frontend/public/index.html (hunk 1 above)
Comment (lappalj4) [reply frontend/public/index.html]: @Earlo can you take care of this? Once it's done I would say this one's safe to merge

**Comment by lappalj4** [comment]
Ok, so as far as I'm concerned this one's good to go after the above mentioned .env changes to our environments.

@biharygergo I changed the logos to use `REACT_APP_LOGO_DARK_URL` and `REACT_APP_LOGO_LIGHT_URL`, so make sure to update the .env for your deployment before pulling these changes :)

File: frontend/public/index.html (hunk 1 above)
Comment (Earlo) [reply frontend/public/index.html]: I see no problems with this

**Pull request merged**